perl-lib/OESS/lib/OESS/Database.pm

```diff
@@ -10099,6 +10099,17 @@ sub add_edge_interface_move_maintenance {
         }
     }
 
+    # move vrf endpoints to new interface
+    my $rec = $self->move_vrf_endpoints(
```

Comment:
is this suppose to be just move_endpoints instead of move_vrf endpoints since it seems we are moving both circuit and vrf endpoints?

Reply:
My understanding is that move_edge_interface_circuits (which is called in the same scope) moves the circuit endpoints. Move_vrf_endpoints only moves entries from the vrf table.

Reply:
ok this is closer... I think i was going for this doing more of an edit where it updated the Endpoint object's interface